yarn.lock

```diff
@@ -3764,6 +3764,7 @@
 
 "@kbn/ai-assistant-icon@link:x-pack/platform/packages/shared/ai-assistant/icon":
   version "0.0.0"
+  uid ""
```

Comment:
Merge conflict error from https://github.com/elastic/kibana/pull/233874. This is correct.